aurora_module/variables.tf

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2011 - 2019, Coveo Solutions Inc.
+ */
+
+// Required variables :
+
+variable "master_username" {
+  description = "Master DB username"
+}
+
+variable "master_password" {
+  description = "Password for the master DB user"
+}
+
+variable "subnet_ids" {
+  type        = "list"
+  description = "A list of VPC subnets IDs"
+}
+
+variable "vpc_id" {
+  type = "string"
+  description = "VPC ID"
+}
+
+// Optional variables :
+
+variable "instances_count" {
+  description = "Number of instances"
+  default = 1
+}
+
+variable "vpc_security_group_ids" {
+  type = "list"
+  default = []
+}
+
+variable "store_master_creds_to_ssm" {
+  default = false
+}
+
+variable "use_secretmanager" {
```

Comment:
Since above we declared `store_master_creds_to_ssm`, I think this variable could be named `store_master_creds_to_secretmanager`. What do you think?

Reply:
I agree. Better naming standards. 😄